### PR TITLE
Fetch the API

### PR DESCRIPTION
**Pull request by Yulianav09**
### Fetch the API

- Fetch a quote from the API.
- Add states for loading and error.

### CODE REVIEW

File: src/components/quote.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+const apiKey = 'MjFx93DKQY9OcA2CcwGaFg==97fQaXcDcvJR7aL3';
+const apiUrl = 'https://api.api-ninjas.com/v1/quotes';
+
+const Quote = () => {
+  const [data, setData] = useState({ quote: 'this is a quote', author: null });
+
+  const getQuote = async () => {
+    setData({ quote: 'Loading...', author: null });
```
Comment (gemmen29) [inline src/components/quote.js]: - [ ] Good job on showing quotes loading but, Kindly create a separate state for loading and use it instead in order to make the `data` state only responsible for storing quote data.

### Hint
```javascript
const [isLoading, setIsLoading] = useState(false)
```

File: src/components/quote.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    try {
+      const response = await fetch(apiUrl, {
+        method: 'GET',
+        headers: { 'X-Api-Key': apiKey },
+      });
+      const responseJson = await response.json();
+      setData(responseJson[0]);
+    } catch (error) {
+      setData({ quote: 'Error fetching the Quote from API', author: null });
```
Comment (gemmen29) [inline src/components/quote.js]: - [ ]  Kindly also create a separate state for error and use it instead in order to make the `data` state only responsible for storing quote data.

### Hint
```javascript
const [hasError, setHasError] = useState(false)
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Yulianav09, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/quote.js (hunk 2 above)
Comment (Yulianav09) [reply src/components/quote.js]: Thanks for the suggestion, I already implemented

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Yulianav09 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**